docs/Prelude.md

```diff
@@ -983,4 +983,20 @@ instance showArray :: (Show a) => Show (Array a)
 instance showOrdering :: Show Ordering
 ```
 
+#### `NaturalTransformation`
+
+``` purescript
+type NaturalTransformation f g = forall a. f a -> g a
+```
+
+A natural transformation
+
+#### `Natural`
+
+``` purescript
+type Natural f g = NaturalTransformation f g
```

Comment:
For a shorter name I think `Transform` might be good. I agree that it would be nice to have `Natural` available for some numerics thing at some point. I like `Transform` for two reasons: 1) the "natural" part is kind of superfluous in my opinion since nobody talks about unnatural transformations and 2) there's a precedent for something similar sounding with [(n,k)-transfors](http://ncatlab.org/nlab/show/transfor#special_cases), which are sometimes used to describe the sequence of functors, natural transformations, …. Homotopy would be another nice short name but probably harder to justify and maybe a bit obscure in a programming context.